From 889efbafc6f831713422884864a1b0adb6e30f38 Mon Sep 17 00:00:00 2001
From: Yashwanth Yadavalli <yashwanth.yadavalli@protonmail.com>
Date: Thu, 21 Jun 2018 18:45:10 -0400
Subject: [PATCH] Move non-ensime specific functions to sbt-mode

---
 sbt-mode.el | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/sbt-mode.el b/sbt-mode.el
index 943e40f..6c4d08c 100644
--- a/sbt-mode.el
+++ b/sbt-mode.el
@@ -92,6 +92,31 @@ sbt:default-command, if no other command has yet been run)."
   (interactive)
   (sbt:command (sbt:get-previous-command)))
 
+(defun sbt-do-compile ()
+  "Compile all sources including tests."
+  (interactive)
+  (sbt:command "test:compile"))
+
+(defun sbt-do-run ()
+  "Execute the sbt `run' command for the project."
+  (interactive)
+  (sbt:command "run"))
+
+(defun sbt-do-test ()
+  "Run all the tests."
+  (interactive)
+  (sbt-command "test"))
+
+(defun sbt-do-clean ()
+  "Execute the sbt `clean' command for the project."
+  (interactive)
+  (sbt:command "clean"))
+
+(defun sbt-do-package ()
+  "Build a jar file of the project."
+  (interactive)
+  (sbt:command "package"))
+
 (defun sbt-completion-at-point ()
   "Complete the command at point. Works both in sbt shell and
 scala console."
