From e9aa908d1b80dc2618eab22eeefc68ae82d0026f Mon Sep 17 00:00:00 2001
From: dickmao <rchiang@cs.stonybrook.edu>
Date: Thu, 31 May 2018 13:10:22 -0400
Subject: [PATCH] address a compiler warning

---
 sbt-mode.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sbt-mode.el b/sbt-mode.el
index 943e40f..682730a 100644
--- a/sbt-mode.el
+++ b/sbt-mode.el
@@ -83,7 +83,7 @@ that outputs errors."
     (setq sbt:previous-command command)))
 
 (defun sbt:get-sbt-completions-for-command (input)
-  (ignore-errors (with-current-buffer (sbt:buffer-name) (sbt:get-sbt-completions input))))
+  (ignore-errors (with-current-buffer (sbt:buffer-name) (sbt:get-completions input))))
 
 ;;;###autoload
 (defun sbt-run-previous-command ()
