From 0ff192089770422eb87f752b9516cfe221d9bbe5 Mon Sep 17 00:00:00 2001
From: "Ross A. Baker" <ross@rossabaker.com>
Date: Fri, 26 Jul 2019 11:06:26 -0400
Subject: [PATCH] Fix live update on files in subdirectories

---
 git-gutter.el | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/git-gutter.el b/git-gutter.el
index 21148b9..aa91d3f 100644
--- a/git-gutter.el
+++ b/git-gutter.el
@@ -1014,9 +1014,10 @@ start revision."
                (buffer-modified-p)
                (git-gutter:should-update-p))
       (let ((file (file-name-nondirectory it))
+            (root (file-truename (vc-git-root it)))
             (now (make-temp-file "git-gutter-cur"))
             (original (make-temp-file "git-gutter-orig")))
-        (when (git-gutter:write-original-content original file)
+        (when (git-gutter:write-original-content original (file-relative-name it root))
           (git-gutter:write-current-content now)
           (git-gutter:start-live-update file original now))))))
 
-- 
2.22.0

